android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1570,6 +1571,7 @@ private void setupWalletModel() {
         setupObservers();
     }
 
+    @MainThread
```

Comment:
nit: no harm to keep this but isn't all method in activity being called on the main thread?

Reply:
Yes, all the activity callbacks are called on the Ui thread. 
Sometimes the IDE gives you a warning if you call a method marked with the `@MainThread` annotation from a background thread. In this specific case is mostly for us, to remember that Ui thread is mandatory, it didn't change much.